utils.js

```diff
@@ -6034,8 +6068,8 @@ export const _swiftDepPkgList = (
     for (const adep of jsonData.dependencies) {
       const urlOrPath = adep.url || adep.path;
       const apkg = {
-        group: adep.identity || "",
-        name: adep.name,
+        group: "",
```

Comment:
This is interesting. I didn't realize that we had a bug like this where we used a group that was not the namespace. As per purl, namespace is the host or user/organization and is mandatory.

https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#swift

Can we parse the URL and use that to construct the group?

Reply:
Is there a way to generate purl from a url without doing anything custom?

Reply:
@prabhu could you review the code and the results if they are ok now?